WordPress/Classes/System/WordPressAppDelegate+Swift.swift

```diff
@@ -5,6 +5,10 @@ import CocoaLumberjack
 // MARK: - Helpers
 
 extension WordPressAppDelegate {
+    @objc func configureNoticePresenter() {
+        let _ = NoticePresenter.shared
```

Comment:
We can (I'd prefer that too), but I don't _think_ we can easily until the entire App Delegate is in Swift? I don't really want to make the presenter an `NSObject`, so the reference can't be in the obj-c App Delegate code, and the Swift code is an extension so can't add properties.

Reply:
Right 😬 I don't mind temporarily making it a `NSObject`, but maybe you can put it inside `StoreContainer` for now?

Reply:
After discussing this in Slack, I'd like to leave this shared for now as the other options have some caveats:

* Making the presenter conform to `NSObject` is tricky because it has non-objective-C properties and an initialiser with a default value.
* Adding it to the StoreContainer is _okay_, but it doesn't really belong there either, and I'd still need to add some code to ensure the StoreContainer is instantiated.

I'm going to add a comment to the code clarifying that it'll move in the near future, and once I add the proper UI the presenter will probably need to be instantiated with a reference to a presenting container – at which point it'll make more sense to create it in the app delegate.